src/test/java/com/unstoppabledomains/resolution/ResolutionTest.java

```diff
@@ -311,25 +312,27 @@ public void buildingDefaultProvider() throws Exception {
 
     @Test
     public void testGetMultiChainAddress() throws Exception {
-        String domainWithMultiChainRecords = "udtestdev-usdt.crypto";
+        String domainWithMultiChainRecords = "testing.crypto";
 
         String erc20 = resolution.getMultiChainAddress(domainWithMultiChainRecords, "usdt", "erc20");
-        assertEquals("0xe7474D07fD2FA286e7e0aa23cd107F8379085037", erc20);
+        assertEquals("0x58cA45E932a88b2E7D0130712B3AA9fB7c5781e2", erc20);
         String tron = resolution.getMultiChainAddress(domainWithMultiChainRecords, "usdt", "tron");
-        assertEquals("TNemhXhpX7MwzZJa3oXvfCjo5pEeXrfN2h", tron);
+        assertEquals("TRMJfXXbmwb3WFSRKbeRgKsYoD8o1a9xxV", tron);
         String omni = resolution.getMultiChainAddress(domainWithMultiChainRecords, "usdt", "omni");
-        assertEquals("19o6LvAdCPkjLi83VsjrCsmvQZUirT4KXJ", omni);
+        assertEquals("1KvzMF2Vjy14d6JGY7dG7vjT5kfpmzSQXM", omni);
+        String eos = resolution.getMultiChainAddress(domainWithMultiChainRecords, "usdt", "eos");
+        assertEquals("karaarishmen", eos);
 
         
     }
 
-    @Test
```

Comment:
Can we test this or remove it completely?

Reply:
Everything else is good to me 👍🏼

Reply:
@sudoryan TokenURIs are different in testnet, but they should be working now. See https://github.com/unstoppabledomains/resolution/blob/3e43971133285c2832fba6004e12349b65cb4ba1/src/tests/Uns.test.ts#L692